README.md

```diff
@@ -35,48 +35,60 @@
     </a>
 </p>
 
-[Haystack](https://haystack.deepset.ai) is an end-to-end framework that enables you to build powerful and production-ready pipelines for different search use cases.
-Whether you want to perform question answering (QA) or semantic document search, you can use the state-of-the-art NLP models in Haystack to provide unique search experiences and allow your users to query in natural language.
-Haystack is built in a modular fashion so that you can combine the best technology from other open source projects, like Hugging Face's transformers, Elasticsearch, or Milvus.
+[Haystack](https://haystack.deepset.ai/) is an end-to-end NLP framework that enables you to build NLP applications powered by LLMs, Transformer models, vector search and more. Whether you want to perform question answering, answer generation, semantic document search, or build tools that are capable of complex decision making and query resolution, you can use the state-of-the-art NLP models with Haystack to build end-to-end NLP applications solving your use case
 
-<p align="center"><img src="https://raw.githubusercontent.com/deepset-ai/haystack/main/docs/img/main_example.gif"></p>
+## Core Concepts
+
+🏃‍♀️ **[Pipelines](https://docs.haystack.deepset.ai/docs/pipelines):** This is the standard Haystack structure that can connect to your data and perform an NLP task on it, that you define. ****A Pipeline is a structure where data flows from one Node to the next where you define how Nodes interact with each other, and how one Node pushes data to the next.
+
+An example pipeline would consist of one `Retriever` Node and one `Reader` Node. When the pipeline is run with a query, the Retriever will first retrieve the relevant documents to the query, and then, the Reader will extract a final answer.
```

Comment:
Can we get rid of the passive: When you run the pipeline with a query... or When the pipeline runs with a query..

Then: .... the Retriever first retrieves the documents relevant to the query and then the Reader extracts the final answer.